src/.vuepress/theme/components/Newsletter.vue

```diff
@@ -17,18 +17,18 @@
           name="member[email]"
           id="member_email"
           required
-          placeholder="Email address"
+          placeholder="Dirección de Correo Electrónico"
         >
         <input
           class="button newsletter-button"
           type="submit"
-          value="Subscribe"
+          value="Suscribirse"
           name="member[subscribe]"
           id="member_submit"
         >
       </form>
       <p>
-        You can read the previous issues and listen to our podcast at <a href="https://news.vuejs.org" target="_blank" rel="noopener">news.vuejs.org</a>.
+        Usted puede leer los problemas previos y escuchar nuestro podcast en <a href="https://news.vuejs.org" target="_blank" rel="noopener">news.vuejs.org</a>.
```

Comment:
I'm not sure if the word "issues" should be translated as "problemas". Although it's the , the word "problemas" gives a different context to the sentence. Also, in the current translate for vuejs2, they translated "issues" as a "números" for this context: https://es.vuejs.org/index.html

Reply:
You're right, @migsalazar! Fixed it.